src/components/tables/RowActions/Delete/Button.tsx

```diff
@@ -14,9 +15,28 @@ interface Props {
 function DeleteButton({ selectableTableStoreName }: Props) {
     const generator = () => null;
 
+    const isCapture =
+        selectableTableStoreName === SelectTableStoreNames.CAPTURE;
```

Comment:
This is okay for now... but I think we should eventually move some setting into the store that controls all this. It could be very direct like `enableRemoveCollectionsOnDelete` or even take it further and just allow any action of some kind of secondary action... however not really sure making it that flexible would really get us anything.